scripts/zones/Bibiki_Bay/IDs.lua

```diff
@@ -43,6 +43,7 @@ zones[xi.zone.BIBIKI_BAY] =
         HAVE_BILLET                   = 7497, -- You cannot buy morrre than one <item>. Use the one you have now to ride the next ship.
         LEFT_BILLET                   = 7502, -- You use your <item>. (<number> trip[/s] remaining)
         END_BILLET                    = 7503, -- You use up your <item>.
+        MTICKET_ADDED                 = 7504, -- Ticket # was/were added to your xxxx
```

Comment:
Weren't you fixing something that was wrong with the manaclipper ticket NPC? Or am I misremembering.

Reply:
Yes will be in another PR. This can stay anyway, needs to go in.